docker/root/init.php

```diff
@@ -11,3 +15,7 @@
     unlink('/root/dbase.config.php');
     unlink('/root/.php-cs-fixer.php');
 }
+
+function setDbEnvParams($envName) {
+    file_put_contents('/root/dbase.config.php', str_replace($envName, getenv($envName), file_get_contents('/root/dbase.config.php')));
+}
```

Comment:
```suggestion
const DB_ENV_PARAMS = ['MYSQL_DATABASE', 'MYSQL_USER', 'MYSQL_PASSWORD'];

function installDbConfig() {
    $config = file_get_contents('/root/dbase.config.php');

    foreach (DB_ENV_PARAMS as $envName) {
        $config = str_replace($envName, getenv($envName), $config);
    }

    file_put_contents('/hleb/database/dbase.config.php', $config);
    unlink('/root/dbase.config.php');
}
```
Я бы предпочёл, чтобы объявление функции было в начале файла. По принципу "сначала объявили, потом используем". Во-вторых, я её немного переписал: стало меньше обращений к файловой системе